x/distribution/module.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // ConsensusVersion defines the current x/distribution module consensus version.
-const ConsensusVersion = 5
+const ConsensusVersion = 4
```

Comment:
why remove this migration ?

Reply:
Because v0.50 is on 3, so there wasn't a need to bump to 5.